server/models/StageModel.js

```diff
@@ -1,12 +1,8 @@
 const mongoose = require('mongoose')
 
 const stageSchema = new mongoose.Schema({
-  _id: {
-    type: String,
-    required: true,
-  },
   tripId: {
-    type: String,
+    type: mongoose.Schema.Types.ObjectId,
     required: true,
```

Comment:
i dont think we will necessary need this for our app, but it's nice to add this (allow usage of .populate(), and the collection actually knows about the relationship):
```
  tripId: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Trip',
    required: true,
  },
```